src/site/blocks/alert.drupal.liquid

```diff
@@ -0,0 +1,45 @@
+{% comment %}
+    Example data:
+    {
+        "entityBundle": "alert",
+        "fieldAlertType": "warning",
+        "fieldHelpText": null,
+        "fieldAlertTitle": "How do I talk to someone right now?",
+        "fieldReusability": "reusable",
+        "fieldAlertContent": {
+            "entity": {
+                "fieldWysiwyg": {
+                "processed": "..."
+                },
+                "fieldTextExpander": "Find out how to get support anytime day or night."
+            }
+        }
+    }
+{% endcomment %}
+{% if alert.fieldAlertContent %}
+    {% assign expander = alert.fieldAlertContent.entity %}
+{% endif %}
+
+<div class="usa-alert usa-alert-{{ alert.fieldAlertType }}" role="alert">
+    <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">
+            {{ alert.fieldAlertTitle }}
+            {% if expander.fieldTextExpander %}
+                <br><a id="crisis-expander-link">{{ expander.fieldTextExpander }}</a>
```

Comment:
This should be a button rather than a link. You can add a class of `va-button-link` to style it as a link.

Reply:
I think we also want to put `aria-expanded="false"` on the button.